### PR TITLE
Update endpoint-dlp-getting-started.md

### PR DESCRIPTION
**Pull request by amirsc3**
Adding important note to pre-requisites

### CODE REVIEW

**Comment by nihendle** [comment]
@chrfox can you please review these changes?

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/endpoint-dlp-getting-started.md [hunk 1]
```diff
@@ -97,7 +97,10 @@ Make sure that the Windows 10 devices that you plan on deploying Endpoint DLP to
 	- For Windows 10 2004 - KB4568831, KB4577063
 	- For devices running Office 2016 (and not any other Office version) - KB4577063 
 
-4. All devices must be [Azure Active Directory (Azure AD) joined](/azure/active-directory/devices/concept-azure-ad-join), AD joined, Hybrid Azure AD joined, or AAD registered.
+4. All devices must be [Azure Active Directory (Azure AD) joined](/azure/active-directory/devices/concept-azure-ad-join), [Hybrid Azure AD joined](/azure/active-directory/devices/concept-azure-ad-join-hybrid), or [AAD registered](/azure/active-directory/user-help/user-help-register-device-on-network).
+   > [!NOTE]
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/endpoint-dlp-getting-started.md:101]:
```suggestion
> [!NOTE] 
```

File: microsoft-365/compliance/endpoint-dlp-getting-started.md [hunk 2]
```diff
@@ -97,7 +97,10 @@ Make sure that the Windows 10 devices that you plan on deploying Endpoint DLP to
 	- For Windows 10 2004 - KB4568831, KB4577063
 	- For devices running Office 2016 (and not any other Office version) - KB4577063 
 
-4. All devices must be [Azure Active Directory (Azure AD) joined](/azure/active-directory/devices/concept-azure-ad-join), AD joined, Hybrid Azure AD joined, or AAD registered.
+4. All devices must be [Azure Active Directory (Azure AD) joined](/azure/active-directory/devices/concept-azure-ad-join), [Hybrid Azure AD joined](/azure/active-directory/devices/concept-azure-ad-join-hybrid), or [AAD registered](/azure/active-directory/user-help/user-help-register-device-on-network).
+   > [!NOTE] 
+   > If using 'AAD registered' devices, then you must reboot the device after completing the AAD registration. This is required for each new user logging-in to the device.<br>
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/endpoint-dlp-getting-started.md:102]:
```suggestion
   > - If you are using _Azure AD registered_ devices, you must reboot the device after completing the Azure AD registration. This is required for each new user signing in to the device.
   > - If you are using either _Azure AD registered_ or _Hybrid Azure AD joined_ devices, the device can be joined to on-premises Active Directory as well.
```

File: microsoft-365/compliance/endpoint-dlp-getting-started.md [hunk 3]
```diff
@@ -97,7 +97,10 @@ Make sure that the Windows 10 devices that you plan on deploying Endpoint DLP to
...
+   > If using either 'AAD registered' or 'Hybrid Azure AD joined' devices, then the device can be joined to on-premises Active Directory as well.<br>
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/endpoint-dlp-getting-started.md]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review. @amirsc3, could you please commit my suggested changes if you agree with them? Many thanks.

**Comment by amirsc3** [comment]
@chrfox can you please confirm that the Azure AD public doc we are pointing to in the AAD registered scenario was updated to include a reboot step? 
If yes, then we should remove the remaining note below:
> [!NOTE] 
   > If using 'AAD registered' devices, then you must reboot the device after completing the AAD registration. This is required for each new user logging-in to the device.<br>

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by chrfox** [comment]
edits made in different PR

**Pull request closed without merging**